jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -177,13 +177,13 @@ public static void SnapDump() {
     	SnapDumpImpl();
     }
     
-	private static native int SnapDumpImpl();
-    
 	private static final DumpPermission DUMP_PERMISSION = new DumpPermission();
 	private static final ToolDumpPermission TOOL_DUMP_PERMISSION = new ToolDumpPermission();
 	private static final String LEGACY_DUMP_PERMISSION_PROPERTY = "com.ibm.jvm.enableLegacyDumpSecurity"; //$NON-NLS-1$
 	
-    private static final class DumpOptionsLock {}
+	static final class DumpOptionsLock {
```

Comment:
The class should remain `private`; add a package-private constructor instead.

Reply:
Seems reasonable.  The compiler was doing that implicitly anyway, right?

Reply:
If no constructor is explicitly defined, the compiler generates a no-argument constructor with the same visibility as the class. A private constructor also implies an accessor function for the use here; hence the motivation for an explicit package-private constructor.